docs/api/extensions.md

```diff
@@ -1,104 +1,112 @@
-# Chrome Extension Support
```

Comment:
Line ending has been changed in this file.

Reply:
@zcbenz Seems like the pre-commit hook changes LF to CRLF automatically

Reply:
I added a change to make sure the files have correct line endings (https://github.com/electron/electron/pull/25134), can you rebase this branch on master?

Reply:
Done!

Reply:
@sentialx Sorry the CI is failing because of forcing line endings, do you mind rebasing to latest master again?

Reply:
Sure